src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,94 @@
+import DropdownMenu from "./DropdownMenu/DropdownMenu";
+import DropdownMenuItem, { MenuItem } from "./DropdownMenuItem/DropdownMenuItem";
+
+export interface DropdownProps {
+    menuItems: MenuItem[][];
+    onChange: (id?: string) => void;
+    activeItemId?: string;
+    placeholder?: string;
+    size?: Size.Small | Size.Large;
+    disabled?: boolean;
+    clearable?: boolean;
+}
+
+const getActiveItem = (menuItems: MenuItem[][], id: string) => menuItems.flat().find((item) => item.id === id) || null;
+
+export default function Dropdown({
+    menuItems,
+    onChange,
+    activeItemId = "",
+    placeholder = "Select item",
+    size = Size.Small,
+    disabled = false,
+    clearable = false,
+}: DropdownProps): ReactElement<DropdownProps> {
+    const [open, setOpen] = useState(false);
+    const activeItem = getActiveItem(menuItems, activeItemId);
+    const wrapperClassNames = [
+        css["trigger-wrapper"],
+        size === Size.Large ? css.large : "",
+        disabled ? css.disabled : "",
+    ].join(" ");
+    const triggerClassNames = [css.trigger, !activeItem ? css.placeholder : "", disabled ? css.disabled : ""].join(" ");
+    const clearButtonClassNames = [css.clear, disabled ? css.disabled : ""].join(" ");
+
+    return (
+        <div className={css.dropdown}>
+            <div data-test-id="dropdown" className={wrapperClassNames}>
+                <button
+                    data-test-id="dropdown-trigger"
+                    onClick={() => setOpen(!open)}
+                    className={triggerClassNames}
+                    disabled={disabled}
+                >
+                    {activeItem ? (
+                        <DropdownMenuItem
+                            title={activeItem.title}
+                            icon={activeItem.icon}
+                            subtitle={activeItem.subtitle}
+                            size={activeItem.size}
+                            style={activeItem.style}
+                            disabled={disabled}
+                            active
+                        />
+                    ) : (
+                        <DropdownMenuItem title={placeholder} size={size} disabled={disabled} />
+                    )}
+                </button>
+                {clearable && activeItem && (
+                    <button
+                        data-test-id="dropdown-clear-button"
+                        className={clearButtonClassNames}
+                        onClick={() => onChange("")}
+                        disabled={disabled}
+                    >
+                        <IconReject size={IconSize.Size13} />
+                    </button>
+                )}
+                <button className={disabled ? css.disabled : ""} onClick={() => setOpen(!open)} disabled={disabled}>
+                    {open ? <IconCaretUp size={IconSize.Size16} /> : <IconCaretDown size={IconSize.Size16} />}
+                </button>
+            </div>
```

Comment:
We miss the contextual options part

<img width="50" alt="Capture d’écran 2021-07-15 à 16 45 37" src="https://user-images.githubusercontent.com/6846950/125807882-47f7f26a-ec81-4bc1-b07c-9c289239d8e8.png">

Reply:
since atm we do not actually know what should happen if someone click on the dots i would add them later in a seperate PR wdyt?

Reply:
I agree with Dominique and have also talked to Simone about it. Maybe we can make a note somewhere to make sure it won't be overlooked when the design is ready.